src/vm/compile.cpp

```diff
@@ -1866,6 +1873,15 @@ BOOL CEEPreloader::DoesMethodNeedRestoringBeforePrestubIsRun(
     return FALSE;
 }
 
+BOOL CEECompileInfo::IsNativeCallableMethod(CORINFO_METHOD_HANDLE handle)
+{
+#ifdef FEATURE_CORECLR
```

Comment:
this ifdef is not needed - you can just piggy back on ifdef in HasNativeCallable()

GetMethod(..) is just a cast that the compiler will optimize away